django/middleware/csrf.py

```diff
@@ -63,21 +101,28 @@ def rotate_token(request):
     """
     request.META.update({
         "CSRF_COOKIE_USED": True,
-        "CSRF_COOKIE": _get_new_csrf_key(),
+        "CSRF_COOKIE": _get_new_csrf_token(),
     })
 
 
 def _sanitize_token(token):
     # Allow only alphanum
-    if len(token) > CSRF_KEY_LENGTH:
-        return _get_new_csrf_key()
```

Comment:
Accepted. The separation of concerns is still correct IMO, but not crucial, and my "became clear" comment above refers to, essentially, optimizing an error case.